src/rustup-cli/rustup_mode.rs

```diff
@@ -702,67 +702,72 @@ fn show(cfg: &Cfg) -> Result<()> {
             print_header("installed toolchains")
         }
         let default_name = cfg.get_default()?;
-        for t in installed_toolchains {
-            if default_name == t {
-                println!("{} (default)", t);
+        let mut t = term2::stdout();
+        for it in installed_toolchains {
+            if default_name == it {
+                let _ = writeln!(t, "{} (default)", it);
```

Comment:
Shouldn't we be using `?` rather than ignoring the error returns from `writeln!()` since this is just informational and so the jump out shouldn't affect matters?